src/io.rs

```diff
@@ -86,12 +86,12 @@ mod tests {
     #[test]
     #[should_panic(expected = "Failed to open file at /made/up/file/location")]
     fn missing_file_causes_a_panic() {
-        files(vec!["/made/up/file/location".to_string()]).unwrap();
+        process_files(vec!["/made/up/file/location".to_string()]).unwrap();
```

Comment:
super-nit but this _could_ pass. Can we make it truly unpassable?

Reply:
> super-nit but this _could_ pass. Can we make it truly unpassable?

but how? an illegal path?

Reply:
Pretty sure it will never come up.

'/dev/null/no/such/path'

Could work?

Reply:
ok I will merge this PR and fix in a future merge